python/Ganga/test/GPI/TestSJSubmit.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+class TestSJSubmit(GangaUnitTest):
+
+    n_subjobs = 10
+
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestSJSubmit, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('Output', 'FailJobIfNoOutputMatched', 'True')
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
```

Comment:
This test doesn't seem to be actually testing the flush/load mechanism. As such I think it would be better (as well as faster) if we avoided doing the full Ganga unload/load between each test function. Perhaps they could all be combined into one test function or maybe use a pytest fixture with module scope.